docs/guides/platform.rst

```diff
@@ -11,12 +11,13 @@ These guides will help you customize or tune aspects of Read the Docs.
     build-using-too-many-resources
     canonical
     conda
+    embedding-content
     environment-variables
     feature-flags
     google-analytics
     hiding-a-version
+    search-indexing
```

Comment:
I'm not sure this is a guide, or something more technical and deep. It feels like guides are user-focused and this is definitely developer focused, so probably belongs somewhere else in our docs?

Reply:
yeah, I agree this isn't user focused. What about https://github.com/readthedocs/readthedocs.org/blob/sss-integration-guide/docs/development/index.rst?

Reply:
I just moved it there